tests/all/test_module.rs

```diff
@@ -98,6 +98,7 @@ fn test_module_owned_data_layout_disposed_safely() {
     context.create_module("test");
 }
 
+#[llvm_versions(3.6..9.0)]
 #[test]
 fn test_write_and_load_memory_buffer() {
```

Comment:
We shouldn't need to duplicate test code here. Just add a llvm_versions/cfg to the `create_object_file` call line?